onnxscript/rewriter/fuse_relus_clips_test.py

```diff
@@ -10,7 +10,8 @@
 from onnx_ir.passes.common import onnx_checker, shape_inference
 
 from onnxscript.rewriter import fuse_relus_clips, testing
-from onnxscript.rewriter import pattern as orp
+from onnxscript.rewriter._rewrite_rule import RewriteRule
```

Comment:
I think the test files can directly import from onnxscript.rewriter, and not reference implementation files? onnxscript.rewriter itself should not have a dependence on the test files, so there should be no circular dependence?